PATCH-P0-FIX-UPSTREAM: Use lib_LIBRARIES instead of pkglib_LIBRARIES

Make more sense and is more intuitive when compiling.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: libmysql/Makefile.shared
===================================================================
--- libmysql/Makefile.shared.orig
+++ libmysql/Makefile.shared
@@ -26,7 +26,7 @@ MYSQLBASEdir=			$(prefix)
 ## We'll use CLIENT_EXTRA_LDFLAGS for threaded and non-threaded
 ## until someone complains that they need separate options.
 LDADD =				@CLIENT_EXTRA_LDFLAGS@ $(target)
-pkglib_LTLIBRARIES =		$(target)
+lib_LTLIBRARIES =		$(target)
 
 noinst_PROGRAMS = conf_to_src
 
Index: libmysqld/Makefile.am
===================================================================
--- libmysqld/Makefile.am.orig
+++ libmysqld/Makefile.am
@@ -37,7 +37,7 @@ INCLUDES=		-I$(top_builddir)/include -I$
 			@condition_dependent_plugin_includes@
 
 noinst_LIBRARIES =	libmysqld_int.a
-pkglib_LIBRARIES =	libmysqld.a
+lib_LIBRARIES =	libmysqld.a
 SUBDIRS =		. examples
 libmysqld_sources=	libmysqld.c lib_sql.cc emb_qcache.cc
 libmysqlsources =	errmsg.c get_password.c libmysql.c client.c pack.c \
